packages/@webex/internal-plugin-flag/package.json

```diff
@@ -29,5 +33,12 @@
     "@webex/test-helper-test-users": "workspace:^",
     "@webex/webex-core": "workspace:^",
     "lodash": "^4.17.21"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.17.10",
+    "@webex/babel-config-legacy": "workspace:^",
+    "@webex/eslint-config-legacy": "workspace:^",
+    "eslint": "^8.24.0",
+    "prettier": "^2.7.1"
```

Comment:
does it throw error for prettier , shouldn't the parent module take care of it

Reply:
It does not, we *can* safely remove it here. I will do this.